packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -65,6 +64,12 @@ const CollectionPostContent = dynamic(() =>
   ).then((module) => module.CollectionPostContent),
 );
 
+const PostAuthBanner = dynamic(() =>
+  import(
+    /* webpackChunkName: "lazyPostAuthBanner" */ '@dailydotdev/shared/src/components/auth/PostAuthBanner'
```

Comment:
Just to follow the current convention 👀 
```suggestion
    /* webpackChunkName: "postAuthBanner" */ '@dailydotdev/shared/src/components/auth/PostAuthBanner'
```